### ISSUE TITLE
ListRenderer are not respecting decorator

### ISSUE THREAD

**Issue by mesquitaa** [contentful/rich-text-renderer-java#39]
Hey all, I was trying to render a list with sublist items and found a strange behaviour on [version](https://github.com/contentful/rich-text-renderer-java/releases/tag/1.7.0) 1.7.0. 

Looks like the library is not respecting the decorator's order: 
<img width="312" alt="Screenshot 2023-08-29 at 16 40 58" src="https://github.com/contentful/rich-text-renderer-java/assets/5247560/5beafca6-7bcf-41e7-8343-e251ee221b42">

The app presents completely different:
<img width="285" alt="Screenshot 2023-08-29 at 16 43 03" src="https://github.com/contentful/rich-text-renderer-java/assets/5247560/632736f3-a1fa-449d-9cbb-b98954f8f1a2">

Basically, the sublist items should show values respecting the contentful editor order

1. NumbersDecorator 
1.1 UpperCaseCharacterDecorator
1.2 LowerCaseRomanNumeralsDecorator
1.3 LowerCaseCharacterDecorator
.... 
1.n LowerCaseCharacterDecorator

**Comment by mesquitaa** [comment]
Guys, any new about this one?

**Comment by rafalniski** [comment]
Hey @mesquitaa this should be fixed in `1.9.0`
